.golangci.yml

```diff
@@ -16,7 +16,7 @@ linters:
     - whitespace
     - goimports
     - gosimple
-#    - errcheck
+    - errcheck
```

Comment:
Took the opportunity to update the golangci lint config as well to help with err checks, error messages and as well fix comments.